### PR TITLE
Achievements

### PR DESCRIPTION
**Pull request by sm-ll**
@jrae 

There seems to be a small bug around counting the achievements. Currently, it outputs %count instead. @pezholio and I are a bit swamped so do you think you could take a look at this?

This should be good to merge into Staging after.

### CODE REVIEW

**Comment by sm-ll** [comment]
[Create achievements visual.](https://trello.com/c/NGMCcaMV/117-create-achievements-visual)

File: README.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ DEFAULT_PASSWORD=SomePassword
 * rake db:migrate
 * rake db:seed # creates a test admin user and adds some schemas and documents
 
-Try 'rails s'. Good luck!
+Try 'rails s'. Good luck!!
```
Comment (jrae) [inline README.md]: haha

File: app/helpers/achievement_helper.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      'cv_completed' => 'file-pdf-o',
+      'job_application' => 'paper-plane',
+      'interview' => 'street-view',
+      'placement_volunteering' => 'handshake-o',
+      'training_started' => 'graduation-cap',
+      'course_completed' => 'graduation-cap',
+      'job_start' => 'id-badge',
+      'job_sustained' => 'shield',
+      'boc_completed' => 'money',
```
Comment (jrae) [inline app/helpers/achievement_helper.rb]: I Would probably have put this in the AchievableOption. Could add it as a third variable - :icon.. That way you tie the IDs to the Object and it's dryer.

File: README.md (hunk 1 above)
Comment (sm-ll) [reply README.md]: haha

**Pull request merged**